README.md

```diff
@@ -108,7 +108,7 @@ $ sudo apt install build-essential cmake cmake-data debhelper dbus google-mock \
     libproperties-cpp-dev libprotobuf-dev libsdl2-dev libsdl2-image-dev lxc-dev \
     pkg-config protobuf-compiler python-minimal
 ```
-We recommend Ubuntu 18.04 (bionic) with **GCC 7.x** as your build environment.
+We recommend Ubuntu 20.04 (focal) with **GCC 7.x** as your build environment.
```

Comment:
Do we still recommend gcc 7.x on 20.04?

Reply:
I am using Clang 9.0 (to use clangd). But gcc 7.5.0 is also fine. Gcc 9.3.0 should also be fine. Should I remove compiler recommendation?

Reply:
Removed info about gcc 7.x.